### DISCUSSION TITLE
Update Members section/dashboard

### DISCUSSION THREAD

**Discussion by ronaldbarendse** [umbraco/Umbraco-CMS#11767] [Features and ideas]
After doing some work on the Members section ([moving member properties to a Member Content App](https://github.com/umbraco/Umbraco-CMS/pull/11587)), I noticed the dashboard looks very outdated and doesn't contain much useful information:

![Members dashboard](https://user-images.githubusercontent.com/1051287/146149257-cf97446e-d1ff-4b51-b896-cf96154ecd6d.png)

I think we can remove the current dashboard, as there are no videos being displayed and [new ones are now added on YouTube, instead of Umbraco TV](https://umbraco.com/blog/annual-price-changes-2022/).

We can replace it with the existing 'All Members' overview instead, so it's more in line with other sections (like Users and Media). Besides this, we can add other improvements like:
- Remove the 'All Members' tree node (as it's actually the parent node for all member types, showing an unfiltered overview)
- Only show the dropdown in the 'Create' button (to select the member type) when viewing all types, otherwise directly create the type selected in the tree
- Show 'Is Approved' and groups as columns in the overview
- Add filtering (similar to Users, e.g. Status, Groups) and ordering
- Possibly move member groups to an app (right top bar) to also align with the Users section?
- Allow extensible batch actions (blue bar after selecting members)

**Comment by bjarnef** [comment]
It would be great if we could streamline users and members sections. And now in Umbraco v9 where members are more consistent with users.
https://github.com/umbraco/Umbraco-CMS/issues/4382

Some filter options and bulk actions (e.g. to approve multiple members) would also be very useful in projects with a lot of members.
